duckduckgo/iOS
# remove lock screen widget pixels

Task/Issue URL: https://app.asana.com/0/414235014887631/1205185501564329/f
Tech Design URL:
CC:

**Description**:
Removes Lock Screen widget pixels.

**Steps to test this PR**:
1. Run the app. 
2. Add some Lock Screen widgets
3. Use them, check no pixels are sent related to their usage



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
